api/representation_test.go

```diff
@@ -375,6 +375,64 @@ func TestRequest_oapigenTaskFromConfigTask(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "services_field_to_condition",
+			taskConfig: config.TaskConfig{
+				Services: []string{"api", "web"},
+			},
+			expectedRequest: oapigen.Task{
+				Condition: oapigen.Condition{
+					Services: &oapigen.ServicesCondition{
+						Names:            &[]string{"api", "web"},
+						UseAsModuleInput: config.Bool(true),
+					},
+				},
+			},
+		},
+		{
+			name: "services_field_to_module_input",
+			taskConfig: config.TaskConfig{
+				Services:  []string{"api", "web"},
+				Condition: &config.ScheduleConditionConfig{Cron: config.String("*/10 * * * * * *")},
+			},
+			expectedRequest: oapigen.Task{
+				ModuleInput: &oapigen.ModuleInput{
+					Services: &oapigen.ServicesModuleInput{
+						Names: &[]string{"api", "web"},
+					},
+				},
+				Condition: oapigen.Condition{
+					Schedule: &oapigen.ScheduleCondition{Cron: "*/10 * * * * * *"},
+				},
+			},
+		},
+		{
+			name: "services_field_to_module_input_w_existing_module_input",
+			taskConfig: config.TaskConfig{
+				Services: []string{"api", "web"},
+				ModuleInputs: &config.ModuleInputConfigs{
+					&config.ConsulKVModuleInputConfig{
+						ConsulKVMonitorConfig: config.ConsulKVMonitorConfig{
+							Path: config.String("fake-path"),
+						},
+					},
+				},
+				Condition: &config.ScheduleConditionConfig{Cron: config.String("*/10 * * * * * *")},
+			},
+			expectedRequest: oapigen.Task{
```

Comment:
`expectedRequest` probably doesn't really make sense anymore, maybe just `expectedTask` , `expectedOapigenTask`, `expectedConversion` etc.

Reply:
Yea, that's true. Renamed to `expected` to allow flexibility for future changes. Test tests a function, so actual/expected language seems straightforward